engine/src/main/scala/cromwell/services/package.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell
+
+import java.sql.Timestamp
+
+import cromwell.backend.BackendJobDescriptorKey
+import cromwell.core.WorkflowId
+import cromwell.engine.ExecutionIndex._
+
+
+package object services {
```

Comment:
I'm okay with whatever organization scheme is agreeable. Why don't we chat about what exactly we want and I'll refactor accordingly.